pyproject.toml

```diff
@@ -1,48 +1,56 @@
 [build-system]
-requires = ["setuptools >= 61.0"]
-build-backend = "setuptools.build_meta"
+requires = [ "poetry-core>=1.2.0" ]
+build-backend = "poetry.core.masonry.api"
+# poetry-core 1.8 not available in .fc39. Can upgrade to 1.8.0 at .fc39 EOL
 
-[project.scripts]
-sherlock = "sherlock:main"
+[tool.poetry-version-plugin]
+source = "init"
 
-[project.urls]
-Homepage = "http://sherlock-project.github.io/"
-Repository = "https://github.com/sherlock-project/sherlock.git"
-Issues = "https://github.com/sherlock-project/sherlock/issues"
-
-[project]
+[tool.poetry]
 name = "sherlock-project"
+version = "0" # single source of truth is __version__.py
+description = "Hunt down social media accounts by username across social networks"
+license = "MIT"
 authors = [
-    { name = "Siddharth Dushantha" }
+    "Siddharth Dushantha"
 ]
 maintainers = [
-    { name = "Matheus Felipe" },
-    { name = "Sondre Karlsen Dyrnes" },
-    { name = "Paul Pfeister" }
+    "Paul Pfeister",
+    "Matheus Felipe",
+    "Sondre Karlsen Dyrnes"
 ]
-description = "Hunt down social media accounts by username across social networks"
 readme = "docs/pyproj/README.md"
-# Do not set license to file. Causes issues with rpm packaging for some reason.
-license = {text = "MIT"}
-dynamic = ["dependencies", "version"]
+packages = [ { include = "sherlock"} ]
 keywords = [ "osint", "reconnaissance", "information gathering" ]
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
     "Intended Audience :: Information Technology",
-    "License :: OSI Approved :: MIT License",
     "Natural Language :: English",
     "Operating System :: OS Independent",
     "Programming Language :: Python :: 3",
     "Topic :: Security"
 ]
+homepage = "http://sherlock-project.github.io/"
+repository = "https://github.com/sherlock-project/sherlock.git"
```

Comment:
I've always been torn on this. I like the `.git` when referring to the repo itself (rather than the web page for the repo) for some extra verbosity, but it's very not necessary

I support removal